### PR TITLE
Full Action Cable documentation read through

### PR DESCRIPTION
**Pull request by maclover7**
This PR checks all active Action Cable documentation for typos and other
fixes. It aims to make sure that when Rails 5 is released, that the
Action Cable docs are up to snuff with the other documentation included
with Rails.

[ci skip]

cc @dhh @matthewd

### CODE REVIEW

**Comment by maclover7** [comment]
@dhh, can you take a read through of this? Want to make sure I'm not missing anything here. :)

r? @dhh

File: actioncable/README.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ The client-side needs to setup a consumer instance of this connection. That's do
 App.cable = ActionCable.createConsumer("ws://cable.example.com")
 ```
 
-The ws://cable.example.com address must point to your set of Action Cable servers, and it
+The `ws://cable.example.com` address must point to your  Action Cable server(s), and it
```
Comment (rafaelfranca) [inline actioncable/README.md]: remove the extra space before `Action`

File: actioncable/lib/action_cable/channel/streams.rb [hunk 2]
```diff
@@ -18,8 +18,8 @@ module Channel
     #     end
     #   end
     #
-    # So the subscribers of this channel will get whatever data is put into the, let's say, `comments_for_45` broadcasting as soon as it's put there.
-    # That looks like so from that side of things:
+    # Based on the above example, the subscribers of this channel will get whatever data is put into the, let's say, `comments_for_45` broadcasting as soon as it's put there.
```
Comment (rafaelfranca) [inline actioncable/lib/action_cable/channel/streams.rb]: Break at 100 characters

File: actioncable/lib/action_cable/connection/base.rb [hunk 3]
```diff
@@ -33,14 +33,14 @@ module Connection
...
-    # established for that current_user (and potentially disconnect them if the user was removed from an account). You can declare as many
+    # First, we declare that this connection can be identified by its current_user. This allows us to later be able to find all connections
+    # established for that current_user (and potentially disconnect them). You can declare as many
     # identification indexes as you like. Declaring an identification means that an attr_accessor is automatically set for that key.
     #
     # Second, we rely on the fact that the WebSocket connection is established with the cookies from the domain being sent along. This makes
     # it easy to use signed cookies that were set when logging in via a web interface to authorize the WebSocket connection.
     #
-    # Finally, we add a tag to the connection-specific logger with name of the current user to easily distinguish their messages in the log.
+    # Finally, we add a tag to the connection-specific logger with name of the current_user to easily distinguish their messages in the log.
```
Comment (rafaelfranca) [inline actioncable/lib/action_cable/connection/base.rb]: `current user` is fine here.

File: actioncable/lib/action_cable/server/configuration.rb [hunk 4]
```diff
@@ -1,6 +1,6 @@
 module ActionCable
   module Server
-    # An instance of this configuration object is available via ActionCable.server.config, which allows you to tweak the configuration points
+    # An instance of this configuration object is available via ActionCable.server.config, which allows you to tweak Action Cable configuration
     # in a Rails config initializer.
     class Configuration
```
Comment (rafaelfranca) [inline actioncable/lib/action_cable/server/configuration.rb]: Is this class public API? Why?

Comment (rafaelfranca) [inline actioncable/lib/action_cable/server/configuration.rb]: oh, it need to be otherwise the methods will be not.

**Comment by maclover7** [comment]
@rafaelfranca - updated PR with your comments.

**Comment by maclover7** [comment]
@rafaelfranca, are we ready to roll here? :)

**Comment by maclover7** [comment]
:tada:

**Pull request merged**